package.json

```diff
@@ -71,17 +71,19 @@
     "concurrently": "^6.0.2",
     "cross-env": "^7.0.3",
     "cross-fetch": "^3.1.4",
-    "cypress": "^7.3.0",
+    "cypress": "^7.4.0",
```

Comment:
Cypress GUI was showing an update available so updated it :man_shrugging: why not right?